jboss/container/wildfly/galleon/cloud-galleon-pack/finalize-script/added/src/main/resources/resources/wildfly/scripts/finalize.cli

```diff
@@ -10,6 +10,22 @@ if (outcome == success) of /subsystem=jgroups/stack=tcp/protocol=MPING:read-reso
   /subsystem=jgroups/stack=tcp/protocol=MPING:remove
 end-if
 
+if (outcome == success) of /subsystem=jgroups/stack=udp/protocol=FD_SOCK:read-resource
+ /subsystem=jgroups/stack=udp/protocol=FD_SOCK:write-attribute(name=socket-binding
```

Comment:
Does the parenthesis need to be closed?  Maybe the command parser is not that strict?

Reply:
Not strictly required. I should fix it.